scout/adapter/mongo/case_events.py

```diff
@@ -413,11 +413,73 @@ def request_rerun(self, institute, case, user, link):
         LOG.debug("Case updated")
         return updated_case
 
+    def monitor(self, institute, case, user, link):
+        """Enable rerun monitoring for a case.
+
+        Args:
+             institute (dict): An Institute object
+             case (dict): Case object
+             user (dict): A User object
+             link (str): The url to be used in the event
+
+        Return:
+             updated_case
+        """
+
+        self.create_event(
+            institute=institute,
+            case=case,
+            user=user,
+            link=link,
+            category="case",
+            verb="monitor",
+            subject=case["display_name"],
+        )
+
+        updated_case = self.case_collection.find_one_and_update(
```

Comment:
This works. But one thing that I noticed while working on a recent PR is that in general all case updates go through the [update_case](https://github.com/Clinical-Genomics/scout/blob/c1630a12b53509b779abf7a06fb8666b36842674/scout/adapter/mongo/case.py#L755).

I think the code would be easier to maintain if you update the case in the same way

Reply:
Hm, this is still in the case adapter.. But, sure, the fewer places we touch the db the better.